cmd/virt-chroot/BUILD.bazel

```diff
@@ -5,11 +5,13 @@ go_library(
     srcs = [
         "main.go",
         "selinux.go",
+        "tap-device-maker.go",
     ],
     importpath = "kubevirt.io/kubevirt/cmd/virt-chroot",
     visibility = ["//visibility:private"],
     deps = [
         "//vendor/github.com/opencontainers/selinux/go-selinux:go_default_library",
+        "//vendor/github.com/songgao/water:go_default_library",
```

Comment:
Pushed https://github.com/vishvananda/netlink/pull/558 into netlink.

Let's see if we can get that in, and replace this lib (if not now, on a follow up PR).